### ISSUE TITLE
Misaligned elements on responsive

### ISSUE THREAD

**Issue by josecruz** [CartoDB/cartodb#5037]
## Production

![artboard1](https://cloud.githubusercontent.com/assets/596509/9275761/75c901e0-4297-11e5-8fd5-f2e22a6556ff.png)
![artboard2](https://cloud.githubusercontent.com/assets/596509/9275763/78516178-4297-11e5-937d-45fde6dc27a8.png)

## Designs
![public map - mobile - logged](https://cloud.githubusercontent.com/assets/596509/9275777/878c3276-4297-11e5-8b84-806c6535e8e6.png)
<img width="638" alt="screen shot 2015-08-14 at 15 21 39" src="https://cloud.githubusercontent.com/assets/596509/9275881/35718a76-4298-11e5-8b8a-681b8de99277.png">

Basically we need to update all margins and spacings to match the designs one. You can compare both versions in `public-map-dataset-2.3-alignments`

**Comment by matallo** [comment]
we discussed to remove some separators then added some, should we stick to the design?
